content/get-started/development-environment/content.md

```diff
@@ -76,6 +76,16 @@ To do this we edit the `scripts` in the `package.json` file:
   "watch:sass": "onchange './src/sass/**/*.scss' -- npm run build",
 }
 ```
+and make sure we let our modules know what browsers we are supporting:
```

Comment:
```suggestion
Autoprefixer uses [Browserslist](https://www.npmjs.com/package/browserslist), so you can specify the browsers you want to target with the `browserlist` key in the `package.json` file like so:
```

Reply:
I'm happy to change it but I want to somehow say that `browserslist` is used for a lot of other modules

Reply:
why?